src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests2.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    [CompilerTrait(CompilerFeature.Patterns)]
+    public class PatternMatchingTests2 : PatternMatchingTestBase
+    {
+        [Fact]
+        public void Patterns2_00()
+        {
+            var source =
+@"
+using System;
+class Program
+{
+    public static void Main()
+    {
+        Console.WriteLine(1 is int {} x ? x : -1);
+    }
+}
+";
+            var compilation = CreateCompilationWithMscorlib45(source, options: TestOptions.DebugExe, parseOptions: TestOptions.RegularWithRecursivePatterns);
+            compilation.VerifyDiagnostics(
```

Comment:
You could move the closing brace to the same line (same for the rest of this file).

Reply:
Having it on the next line makes it easier to insert diagnostics if/when needed, and if/when this test is modified or copied.

---
In reply to: [155100159](https://github.com/dotnet/roslyn/pull/23209#discussion_r155100159) [](ancestors = 155100159)